.travis.yml

```diff
@@ -66,10 +56,24 @@ jobs:
         - mvn test -B
     - stage: test
       name: integrationTests
-      script:
+      # To ensure docker containers are fully up and running we sleep 60s
```

Comment:
If we're going with 90, update the comment to reflect this (and maybe move the comment to the sleep command?)